cli/cmd/testdata/install_default.golden

```diff
@@ -398,7 +398,7 @@ spec:
   selector:
     conduit.io/control-plane-component: prometheus
   ports:
-  - name: http
+  - name: admin-http
```

Comment:
I noticed Istio requires people to name ports with the protocol as a prefix, such as `http-admin`. Is that perhaps part of a growing pattern, and should we follow it?

Reply:
Good question. I'd be all for standardizing on something like that.

The change in this part is not quite that deliberate. Our existing Prometheus setup is configured to scrape all control-plane components with an `admin-http` port:
https://github.com/runconduit/conduit/blob/680bf6211ab197af3ed17af968de9ed5b0ccd17b/cli/install/template.go#L349-L352

The Prometheus container itself was the one exception here, so I changed it to match the other control-plane components.